config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -964,8 +964,12 @@ presubmits:
         image: gcr.io/k8s-testimages/kubekins-e2e:v20200802-ff98b58-master
         name: ""
         resources:
+          limits:
+            cpu: 4
+            memory: "13Gi"
```

Comment:
I think we need a bit over 13 here per https://github.com/kubernetes/test-infra/issues/18589

Reply:
ok, i did not fully get that :) will change

Reply:
done